cloud-logging/apiproxy/samples-cloud-logging.xml

```diff
@@ -0,0 +1,16 @@
+<APIProxy revision="1" name="samples-cloud-logging">
+    <DisplayName>Samples Cloud Logging</DisplayName>
```

Comment:
- Include the copyright comment
- Exclude all except Description
```xml
<APIProxy name="sample-cloud-logging>
    <Description>Proxy that writes custom log messages to Cloud Logging</Description>
</APIProxy>
```

Reply:
changed